build.gradle

```diff
@@ -1,7 +1,7 @@
 apply plugin: 'java'
-//apply plugin: 'com.google.protobuf'
+apply plugin: 'com.google.protobuf'
 
-group 'com.spaceuptech.api.core'
+group 'com.spaceuptech.space_api'
```

Comment:
Why did we do this change?

Reply:
That was already updated in the proto file, so I had to do the change.
Also, everywhere we have named the API as `space_api`

Reply:
Got it